plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src/org/wso2/developerstudio/eclipse/gmf/esb/provider/CacheMediatorItemProvider.java

```diff
@@ -86,6 +79,28 @@ public CacheMediatorItemProvider(AdapterFactory adapterFactory) {
 		return itemPropertyDescriptors;
 	}
 
+	/**
+	 * This adds a property descriptor for the Cache Id feature.
+	 * <!-- begin-user-doc -->
+	 * <!-- end-user-doc -->
+	 * @generated
+	 */
+	protected void addCacheIdPropertyDescriptor(Object object) {
```

Comment:
We dont need the cache Id property with new cache mediator implementation. Shall we remove this? https://docs.wso2.com/display/EI620/Cache+Mediator

Reply:
Removed with https://github.com/wso2/devstudio-tooling-esb/pull/381/commits/faf17d41dfdf7c9978e2ce14d164b2682f4caa22 & https://github.com/wso2/devstudio-tooling-esb/pull/381/commits/2b4aecbd52ffa311dc1c05d1fc73c99e8f25d412